libr/io/section.c

```diff
@@ -47,13 +48,32 @@ R_API int r_io_section_exists_for_id (RIO *io, ut32 id)
 	return false;
 }
 
+RIOSection *_section_chk_dup (RIO *io, ut64 addr, ut64 vaddr, ut64 size, ut64 vsize, int flags, const char *name, ut32 bin_id, int fd)
+{
+	RIOSection *sec;
+	SdbListIter *iter;
+	char sname[32];
+	if (!name)
+		snprintf (sname, 31, "section.0x016%"PFMT64x"", vaddr);
+	ls_foreach (io->sections, iter, sec) {
+		if ((sec->addr == addr) && (sec->vaddr == vaddr) && (sec->size == size) &&
```

Comment:
maybe put this conditional in a separate function in camel case with a descriptive name.